src/demo.js

```diff
@@ -137,85 +137,242 @@ document.addEventListener("DOMContentLoaded", () => {
+								category: 'GIS & MAPPING PRODUCTS',
+								cols: [
+									{
+										col: [
+											{
+												heading: 'SOLUTIONS',
+												label: 'Mapping',
+												href: 'https://www.esri.com/en-us/location-intelligence'
+											},
+											{
+												heading: 'GIS & MAPPING PRODUCTS',
+												label: 'ArcGIS Online',
+												href: 'https://www.esri.com/en-us/arcgis-online'
+											},
+											{
+												heading: 'CAPABILITIES',
+												label: 'Mapping',
+												href: 'https://www.esri.com/en-us/location-intelligence'
+											},
+											{
+												heading: 'GIS & MAPPING PRODUCTS',
+												label: 'ArcGIS Online',
+												href: 'https://www.esri.com/en-us/arcgis-online'
+											},
+											{
+												heading: 'CAPABILITIES',
+												label: 'Mapping',
+												href: 'https://www.esri.com/en-us/location-intelligence'
+											},
+											{
+												heading: 'GIS & MAPPING PRODUCTS',
+												label: 'ArcGIS Online',
+												href: 'https://www.esri.com/en-us/arcgis-online'
+											}
+										]
+									}
 								]
 							},
 							{
-								label: 'Smart Communities',
-								href: 'https://www.esri.com/smart-communities',
-								icon: [
-									'M7 32.1h4v.8H7zm24 6.8h3.757v-.8H31zm0 2h3.757v-.8H31zm-24-2h4v-.8H7zm20-10.8h-2v.8h2zM7 35.9h4v-.8H7zm20-10.8h-2v.8h2zM47.4 44c0 2.861-13.994 3.9-23.4 3.9S.6 46.861.6 44c0-.737.805-1.337 2.5-1.879V29.1h3v-4h5.6l3.2 4.267V33.1h3.2v-6.233l1-1.334V15h.01l-1.01-1.094V3.843L21.575.1H27.9v6h2v21h4.2v-4.738l3.33-4.262h5.47v23.498c4.016.868 4.5 1.783 4.5 2.402zm-.8 0c0-.032-.053-.773-3.7-1.584V43h-.8V18.9h-4.28l-2.92 3.738V27.9h-5v5.2h2.2v-2h5.8V44h-.8v-8.1h-3v-2h-7v-2h-4.2V44h-.8V31.1h5.8v2h1.2V6.9h-2v-6h-5.175L18.9 4.157v9.437l3 3.25v6.29l-3 4V44h-.8V33.9h-4v-4.267L11.3 25.9H6.9v4h-3V43h-.8v-.039c-1.322.455-1.7.858-1.7 1.039 0 1.294 8.598 3.1 22.6 3.1s22.6-1.806 22.6-3.1zM21.1 22.867v-5.71l-1.2-1.3v8.609zM34.9 33.1v2h2.2v-3.2h-4.2v1.2zm2.2-5.1h.8v-2.791h-.8zM27 19.1h-2v.8h2zm0 3h-2v.8h2zm0-6h-2v.8h2zm0-6h-2v.8h2zM39.1 28h.8v-5h-.8zM27 13.1h-2v.8h2zm-18.6-.6a.9.9 0 1 0-.9.9.901.901 0 0 0 .9-.9zm29.1-.1a.9.9 0 1 1 .9-.9.901.901 0 0 1-.9.9zm1.267 1.705l.697-.394a17.834 17.834 0 0 1 1.28 2.789h-.854a16.573 16.573 0 0 0-1.123-2.396zM31.5 6.125a18.543 18.543 0 0 1 4.292 2.962l-.541.589a18.317 18.317 0 0 0-1.621-1.327 18.097 18.097 0 0 0-2.13-1.314zM4.107 23.512c-.005-.17-.007-.341-.007-.512a18.864 18.864 0 0 1 1.795-8.05l.725.34a18.05 18.05 0 0 0-1.713 8.199L4.5 23.5zm5.66-12.861l-.584-.546A18.775 18.775 0 0 1 16.5 5.257v.856a17.943 17.943 0 0 0-6.733 4.538z'
+								category: 'LOCATION-ENABLED PRODUCTS',
+								cols: [
+									{
+										col: [
+											{
+												heading: 'CAPABILITIES',
```

Comment:
These headings should not be uppercase. If we wish to display them uppercase we should use `text-transform: uppercase;`. The reason is that some words when written in all uppercase are read as acronyms by screen readers.  Like the word "It" will be read as "I.T.".